<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gctMainWindow</class>
 <widget class="QMainWindow" name="gctMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gctMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>148</width>
         <height>429</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="CoordMenuButton">
          <property name="text">
           <string>Coordinates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LetterMenuButton">
          <property name="text">
           <string>Letters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NumberMenuButton">
          <property name="text">
           <string>Numbers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="CoordPage">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="CoordTabWigdet">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="CoordTab01">
           <attribute name="title">
            <string>Translate</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLineEdit" name="CoordLineDegree"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="CoordButtonDegree">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="CoordLineDegreeMinute"/>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="CoordLineDegreeMinuteSecond"/>
            </item>
            <item row="7" column="0">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="CoordButtonDegreeMinute">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="CoordButtonDegreeMinuteSecond">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="CoordLabelDegree">
              <property name="text">
               <string>Degrees (e.g. 53.087117 8.791817)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QLabel" name="CoordLabelDegreeMinute">
              <property name="text">
               <string>Degrees Minutes (e.g. N 53째 05.227 E 008째 47.509)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="QLabel" name="CoordLabelDegreeMinuteSecond">
              <property name="text">
               <string>Degrees Minutes Seconds (e.g. N53째 5' 13.62&quot; E8째 47' 30.54&quot;)</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CoordTab02">
           <attribute name="title">
            <string>Project</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LetterPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="LetterTabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="LetterTab01">
           <attribute name="title">
            <string>Letters to Numbers</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLineEdit" name="LetterLineInput"/>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="0">
               <widget class="QLabel" name="LetterLabelDirection">
                <property name="text">
                 <string>Direction</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="LetterComboDirection">
                <item>
                 <property name="text">
                  <string>A=1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z=1</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="LetterLabelOffset">
                <property name="text">
                 <string>Offset</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="LetterComboInput">
                <item>
                 <property name="text">
                  <string>Letters to Numbers</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Numbers to Letters</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="LetterLabelInput">
                <property name="text">
                 <string>Input</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="LetterSpinOffset">
                <property name="minimum">
                 <number>-9999</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="LetterLineOutput"/>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="7" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="0">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="LetterLabelCharacters">
                <property name="text">
                 <string>#Characters</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="LetterLineCharacters">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="LetterLabelLetters">
                <property name="text">
                 <string>#Letters</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="LetterLineLetters">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="LetterLineCrossSumLetterSum">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="LetterLabelCrossSumLetterSum">
                <property name="text">
                 <string>Cross sum of Letter sum</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="LetterLabelLetterSum">
                <property name="text">
                 <string>Letter sum</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="LetterLineLetterSum"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="LetterLabelCrossSumLetterSumIterated">
                <property name="text">
                 <string>Cross sum of Letter sum iterated</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="LetterLineCrossSumLetterSumIterated"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="LetterTab02">
           <attribute name="title">
            <string>Letter codes</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="4" column="0">
             <widget class="QLineEdit" name="LetterCodesInputCode"/>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="LetterCodesToLetter">
              <property name="text">
               <string>Code to Letter</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="LetterCodesInputLetter"/>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="LetterCodesToCode">
              <property name="text">
               <string>Letter to Code</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QComboBox" name="LetterCodesSelectCode"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NumberPage">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="NumberTabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="NumberTab01">
           <attribute name="title">
            <string>Numeric Systems</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="NumberTab02">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>gctMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
